### PR TITLE
2.4 Bedfan Backplates

### PR DESCRIPTION
**Pull request by Ferdi360**
### Which mods are added by this PR?
2.4 Bedfan Backplates

### How was it tested?
350 version by myself. Fits nicely. Spacing of the bed extrusions is the same for other sizes. 

### Any background context you want to provide?
Backplates for your Voron 2.4 that block hot air out of your bedfans from blowing onto your ACM panels to (slightly) improve airflow and temps. Also a nice way to showoff Voron branding and the serial of your printer.
I noticed a hotspot on the lower third of the ACM backpanel, which now gets more equally warm. Also the top panel gets warmer sooner, which indicated improved flow for me.

### Screenshots (if appropriate)
2 4 Backplates

### ⚠️⚠️**PLEASE READ AND COMPLETE THE CHECKLIST BELOW**⚠️⚠️

  * [ x] I have read the rules available [here](https://github.com/VoronDesign/VoronUsers/wiki/Mod-Submission-Rules) and
        my mod adheres to these rules.
  * [ x] This mod was created by myself and I agree to publish it under the repository
        [license](https://github.com/VoronDesign/VoronUsers/blob/master/LICENSE.md)

### CODE REVIEW

**Comment by chirpy2605** [comment]
Could you please upload a .metadata.yml file as explained in the submission docs here:
https://github.com/VoronDesign/VoronUsers/wiki/How-to-build-a-metadata.yml-file

This is a required file so that PR can be merged correctly.

**Review by chirpy2605 — CHANGES_REQUESTED** [review]
We've just discovered a larger problem. You've submitted this as a branch from the old "master" target and not the new "main" one:

Details are in this wiki entry:
https://github.com/VoronDesign/VoronUsers/wiki/VoronUsers-Overhaul

> Make sure your new PR targets the main branch. The old master branch will be retired soon and won't receive any updates

I think you're going to need to close this PR and create a new one on the correct branch

File: printer_mods/Ferdi360/2.4 Backplates/CAD/2.4 Backplate v12.f3d
[the remark this answers is not in the document]
Comment (Ferdi360) [reply printer_mods/Ferdi360/2.4 Backplates/CAD/2.4 Backplate v12.f3d:1]: Hi Chirpy, thanks for all your comments (and your patience!). First timer here so have to get used to it a bit. I saw your note about the repository, i will make a new PR when i got the new main one. :)

[the remark this answers is not in the document]
Comment (chirpy2605) [reply printer_mods/Ferdi360/2.4 Backplates/CAD/2.4 Backplate v12.f3d:1]: Thanks. I should have spotted that from the start, but the other bits needed doing anyway. Just remember to backup your files before removing the branch on your fork so you don't lose them.

**Pull request closed without merging**